src/utils/liveReload.ts

```diff
@@ -220,23 +263,32 @@ const applyRichTextElement = (
       links: update.data.links,
       images: update.data.images,
       linkedItems,
-    })
+    }),
   );
 
   return chainOptionallyAsync(withItems, (el) =>
-    applyOnOptionallyAsync(updateComponents(update.data.linkedItems, el.linkedItems), (linkedItems) => ({
-      ...el,
-      linkedItems,
-    }))
+    applyOnOptionallyAsync(
+      updateComponents(update.data.linkedItems, el.linkedItems),
+      (linkedItems) => ({
+        ...el,
+        linkedItems,
+      }),
+    ),
   );
 };
 
 const areItemsSame = (item1: IContentItem, item2: IContentItem): boolean =>
   item1.system.codename === item2.system.codename &&
   item1.system.language === item2.system.language &&
-  Object.entries(item1.elements).every(([codename, el1]) => areElementsSame(el1, item2.elements[codename]));
+  Object.entries(item1.elements).every(([codename, el1]) =>
+    areElementsSame(el1, item2.elements[codename]),
+  );
 
-const areElementsSame = (el1: ElementModels.IElement<any>, el2: ElementModels.IElement<any>): boolean => {
+const areElementsSame = (
+  el1: ElementModels.IElement<any>,
+  el2: ElementModels.IElement<any>,
+): boolean => {
+  // eslint-disable-next-line @typescript-eslint/switch-exhaustiveness-check
```

Comment:
Here the same, I would rather explicitly use case ElementType.Unknown instead of default.